index.js

```diff
@@ -0,0 +1,151 @@
+// task__1
+
+const sayHello = function (name) {
+    if (name == 'Mark') {
```

Comment:
what if I call your function with mark (lower cased first letter)? Improve your function by comparison in one register